template/build/webpack.base.conf.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
           postcss: cssUtils.postcss,
           loaders: merge({js: 'babel-loader'}, cssUtils.styleLoaders({
             sourceMap: useCssSourceMap,
-            extract: env.prod
+            extractCSS: env.prod
```

Comment:
No worries. Please take a look at a barebones starter kit's src/main.js. Notice the difference between `require` and `import`. Your problem should not exist if you use `require` for Quasar's CSS.